src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -89,20 +120,19 @@ def add_cookie(self, name, value, path=None, domain=None, secure=None,
 
 
 class CookieInformation(object):
-    def __init__(self, cookie):
-        self.domain = cookie['domain'] if 'domain' in cookie else None
-        self.expiry = cookie['expiry'] if 'expiry' in cookie else None
-        self.httpOnly = cookie['httpOnly'] if 'httpOnly' in cookie else None
-        self.name = cookie['name'] if 'name' in cookie else None
-        self.path = cookie['path'] if 'path' in cookie else None
-        self.secure = cookie['secure'] if 'secure' in cookie else None
-        self.value = cookie['value'] if 'value' in cookie else None
+    def __init__(self, domain, expiry, httpOnly, name, path, secure, value):
```

Comment:
Isn't there any better order for these arguments than alphabetical? I'd have name and value first, and others could be in same order as here in the spec: https://w3c.github.io/webdriver/webdriver-spec.html#cookies

Should use same order also in the table listing the attributes.

Reply:
Done